tests/installation/add_additional_products.pm

```diff
@@ -12,7 +12,7 @@ use strict;
 use warnings;
 
 sub run {
-    $testapi::distri->get_product_selection()->add_selected_products();
+    $testapi::distri->get_add_on_product()->add_selected_products();
```

Comment:
it would makes more sense to put this test module under `installation/add_on_product_installation/`